devicetypes/sky-nie/min-smart-plug-dimmer.src/min-smart-plug-dimmer.groovy

```diff
@@ -140,17 +145,9 @@ def executeConfigureCmds() {
 
     state.resyncAll = false
     if (cmds) {
-        sendCommands(delayBetween(cmds, 500))
-    }
-    return []
-}
-
-private sendCommands(cmds) {
-    def actions = []
-    cmds?.each {
-        actions << new physicalgraph.device.HubAction(it)
+        def commands = [List, of, cmds]
```

Comment:
What I wrote earlier https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/69768#discussion_r669750343 was an example. It meant that object `cmds` is literally a "List of commands". Not a list containing variables `List` (which, as a side note, is a type, not a variable), `of` (which isn't even declared*) and `commands`.
In your code, you'd just use 
```
sendHubCommand(cmds, 500)
```

\* \- which additionally proves me, that you don't test your code with a device (cause you'd get an exception). Keep in mind that is your responsibility to make sure that device works correctly under your code, and it's your job to maintain it later.

Reply:
Oh, A super low-level, stupid error...